lua/tabline/highlights.lua

```diff
@@ -6,29 +6,33 @@ M.get_color = function(group, attr)
 end
 
 M.c = {
-    active_bg = M.get_color('TabLineSel', 'bg'),
-    inactive_bg = M.get_color('TabLine', 'bg'),
     active_text = '#eeeeee',
     inactive_text = '#7f8490',
     active_sep = '#ff6077',
 }
 
+M.get_bg = function (active)
+    local group = active and 'TabLineSel' or 'TabLine'
+    return M.get_color(group, 'bg')
+end
+
 local hi = function(colors)
     for name, opts in pairs(colors) do
+        opts.default = true
         vim.api.nvim_set_hl(0, name, opts)
     end
 end
 
 -- stylua: ignore start
 hi({
-     TabLineSeparatorSel = { fg = M.c.active_sep,    bg = M.c.active_bg } ,
-     TabLineSeparator    = { fg = M.c.inactive_text, bg = M.c.inactive_bg } ,
-     TabLineModifiedSel  = { fg = M.c.active_text,   bg = M.c.active_bg } ,
-     TabLineModified     = { fg = M.c.inactive_text, bg = M.c.inactive_bg } ,
-     TabLineCloseSel     = { fg = M.c.active_text,   bg = M.c.active_bg } ,
-     TabLineClose        = { fg = M.c.inactive_text, bg = M.c.inactive_bg } ,
-     TabLinePaddingSel   = { link = 'TabLineSel' },
-     TabLinePadding      = { link = 'TabLine' },
+    TabLineSeparatorSel = { fg = M.c.active_sep,    bg = M.c.active_bg } ,
```

Comment:
We should use `M.get_bg` here since you removed the `active_bg` and `inactive_bg` from the colors table. Other than that everything looks good to me. So I will merge after that :)

Reply:
Oh, I missed that. Let me fix this.